src/main/kotlin/org/eclipse/tractusx/bpdmcertificatemanagement/entity/CertificateDB.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.eclipse.tractusx.bpdmcertificatemanagement.entity
+
+import jakarta.persistence.*
+import java.time.LocalDateTime
+import java.util.SortedSet
+
+@Entity
+@Table(name = "certificate")
+class CertificateDB(
```

Comment:
Almost every variable here is a value. You don't plan on updating certificates? If you do, then these should be vars instead

Reply:
Yes i am planning for updating certificates by adding unique identifier in another pull request which Alex is working on currently and that attribute will be called as documentID. Also, made most of attributes to var now but this can change in future once i get more clarity.